dti-website-redesign/src/components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from './Footer.module.css';
+
+type Icon = {
+  link: string;
+  src: string;
+  alt: string;
+};
+
+const icons: Icon[] = [
```

Comment:
Idk if this is too important, but the icons are in a different order from the design.